jobs/e2e_node/crio/templates/crio_cgroupsv2.yaml

```diff
@@ -16,7 +16,7 @@ storage:
       mode: 0644
     - path: /etc/crio/crio.conf.d/20-crio.conf
       contents:
-        local: 20-crio.conf
+        local: 20-crio-v2.conf
```

Comment:
what's the diff between these two? do we need a separate one if we're only updating the commit below?

Reply:
The libexec path usage is new.